### PR TITLE
feat: add new rule related to moveit exploitation

### PR DESCRIPTION
**Pull request by nasbench**

### CODE REVIEW

File: yara/vuln_moveit_0day_jun23.yar [hunk 1]
```diff
@@ -73,3 +73,17 @@ rule LOG_EXPL_MOVEit_Exploitation_Indicator_Jun23_2 {
...
+      description = "Detects a potential compromise indicator found in MOVEit DMZ Web API logs"
+      author = "Nasreddine Bencherchali"
+      reference = "https://attackerkb.com/topics/mXmV0YpC3W/cve-2023-34362/rapid7-analysis"
+      date = "2023-06-13"
+      score = 70
+   strings:
+      $x1 = "TargetInvocationException" ascii
+      $x2 = "MOVEit.DMZ.Application.Folders.ResumableUploadFilePartHandler.DeserializeFileUploadStream" ascii
+   condition:
+      all of ($x*)
```
Comment (phantinuss) [inline yara/vuln_moveit_0day_jun23.yar:88]:
```suggestion
      $s1 = "TargetInvocationException" ascii
      $s2 = "MOVEit.DMZ.Application.Folders.ResumableUploadFilePartHandler.DeserializeFileUploadStream" ascii
   condition:
      all of ($s*)
```

**Review by phantinuss — APPROVED** [review]

**Comment by Neo23x0** [comment]
The problem with "LOG_" rules in the public repo is that not only THOR Lite uses that repo but also LOKI and other people with their YARA scanning engines that wouldn't apply "LOG_" rules in the right way. 
Just recently I had a problem with `LOG_LibSSH_Auth_Bypass_CVE_2023_2283_Jun23_1` that caused many FPs because it matched on many different files, not just log lines in log files.

**Comment by Neo23x0** [comment]
But it looks okay :
Screenshot 2023-06-14 at 08 40 23

**Review by Neo23x0 — COMMENTED** [review]
Looks good

**Pull request merged**